source/base/timer.cc

```diff
@@ -13,16 +13,17 @@
 //
 // ---------------------------------------------------------------------
 
-#include <deal.II/base/timer.h>
 #include <deal.II/base/exceptions.h>
+#include <deal.II/base/timer.h>
 #include <deal.II/base/mpi.h>
-#include <deal.II/base/utilities.h>
 #include <deal.II/base/signaling_nan.h>
-#include <sstream>
+#include <deal.II/base/utilities.h>
+
+#include <algorithm>
+#include <chrono>
 #include <iostream>
 #include <iomanip>
-#include <algorithm>
-#include <stddef.h>
+#include <sstream>
```

Comment:
... and <iomanip> and <iostream>. Or did I miss the point of moving these headers totally?